Stripe/PublicHeaders/STPPaymentMethodBillingDetails.h

```diff
@@ -0,0 +1,47 @@
+//
+//  STPPaymentMethodBillingDetails.h
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 3/5/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPAPIResponseDecodable.h"
+
+@class STPPaymentMethodBillingDetailsAddress;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface STPPaymentMethodBillingDetails : NSObject <STPAPIResponseDecodable>
+
+/**
+ Billing information associated with a `STPPaymentMethod` that may be used or required by particular types of payment methods.
+ 
+ @see https://site-admin.stripe.com/docs/api/payment_methods/object#payment_method_object-billing_details
```

Comment:
let's not include references to internal URLs (here and elsewhere)

Reply:
Is there an alternative I can use?

Reply:
you can drop the site-admin subdomain